templates/_schemas/definitions/organization.twig

```diff
@@ -1,12 +1,25 @@
 {% macro organization(props, schema = null) %}
+  {% from '_schemas/definitions/thing' import thing %}
   
   {% set schema = schema ?? craft.schema.Organization %}
   {% if props.url ?? false %}
     {% set schema = schema.id(props.url ~ '#identity') %}
   {% endif %}
+  {% set schema = thing(props, schema) %}
   {% set schema = schema
     .name(props.title)
```

Comment:
`name` is set by `Thing`, can be removed from here now.

Reply:
Done.